eng/illink.targets

```diff
@@ -192,6 +192,13 @@
       <ILLinkArgs>$(ILLinkArgs) --disable-opt unusedinterfaces</ILLinkArgs>
       <!-- keep DynamicDependencyAttribute unless a project explicitly disables it -->
       <ILLinkArgs Condition="'$(ILLinkKeepDepAttributes)' != 'false'">$(ILLinkArgs) --keep-dep-attributes true</ILLinkArgs>
+      <!-- enable verbose output -->
+      <ILLinkArgs>$(ILLinkArgs) --verbose</ILLinkArgs>
+      <!-- suppress "error IL2025: Duplicate preserve of" warnings from verbose output,
+           as we need both linker annotations and xml files
+           suppress "error IL2047: All overridden members must have the same DynamicallyAccessedMembersAttribute usage",
+           as we don't add suppression files for ref projects -->
+      <ILLinkArgs>$(ILLinkArgs) --nowarn IL2025;IL2047</ILLinkArgs>
```

Comment:
> IL2047

This is a workaround for https://github.com/mono/linker/issues/1348.

Reply:
All the 2047 should be fixed within the runtime. I know @MichalStrehovsky planned to look into it. I think we should have an issue for fixing these in the runtime - or add them to the baselines as well.

mono/linker#1348 might help make the baselines "smaller" if we link against implementation assemlies.